core/common/src/IpcAppProps.ts

```diff
@@ -91,5 +91,11 @@ export interface IpcAppFunctions {
   toggleInteractiveEditingSession: (key: string, _startSession: boolean) => Promise<boolean>;
   /** @internal */
   isInteractiveEditingSupported: (key: string) => Promise<boolean>;
+  /** @internal */

```

Comment:
this entire class is marked @internal, these comments aren't necessary. The documentation should be on the public methods.

Reply:
Oops, just copied from existing methods, didn't look at class. Fixed.